cmd/lncli/commands.go

```diff
@@ -423,6 +429,13 @@ func openChannel(ctx *cli.Context) error {
 		}
 	}
 
+	if ctx.IsSet("announce") {
+		req.Announce = ctx.Bool("announce")
+	} else {
+		// TODO(halseth): make default be false.
+		req.Announce = true
```

Comment:
Note that you can also set the default directly by using the `Value: default` attribute directly on the command's definition.

Reply:
I made it `Announce` since we want it to default to `false` eventually. And I guess we should have the invoice routing hints supported before we do that?

Reply:
And it doesn't look like `BoolFlag` has a `Value` attribute...

Reply:
Well we'd set it to false only in the desktop app or mobile app.

Reply:
Ok, I'll change the flag back to `private`, and make it false by default.